.eslintrc.json

```diff
@@ -1,7 +1,10 @@
 {
   "root": true,
   "extends": ["./node_modules/sanctuary-style/eslint.json"],
-  "parserOptions": {"ecmaVersion": 2020, "sourceType": "module"},
+  "parserOptions": {"ecmaVersion": 2022, "sourceType": "module"},
```

Comment:
This version is required to use [private properties][1] (in the `Config` class). Browser support for this feature is currently at [95%][2]. Shall we use a closure instead?

> And should the node version in the package.json be updated as a result?

Good catch. We should require `>=14.6.0`.


[1]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_properties
[2]: https://caniuse.com/mdn-javascript_classes_private_class_fields

Reply:
We use Node.js to run the test suite because it's so much easier than testing in a browser, but I want the library to work in browsers too.

My understanding is that these days npm is also used for front-end packages.

Reply:
I decided to use a closure instead.